tests/MongoDB.Driver.Core.Tests/ChangeStreamDocumentSerializerTests.cs

```diff
@@ -13,69 +13,53 @@
 * limitations under the License.
 */
 
+using System;
+using System.IO;
 using FluentAssertions;
 using MongoDB.Bson;
 using MongoDB.Bson.IO;
 using MongoDB.Bson.Serialization;
 using MongoDB.Bson.Serialization.Serializers;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using MongoDB.Bson.TestHelpers;
+using MongoDB.Bson.Tests.Serialization.Serializers;
 using Xunit;
 
 namespace MongoDB.Driver
 {
     public class ChangeStreamDocumentSerializerTests
```

Comment:
These tests are rather different than before because the class itself is quite different now.